docs/cluster-deployment/stripe.rst

```diff
@@ -20,134 +20,117 @@ powered by `Stripe`_.
+   :alt: Region selection
 
-Now comes the stage to finalize the subscription and configure your soon-to-be-
-deployed cluster. The next part will take you through the CrateDB Cloud
-cluster configuration wizard. In four quick steps, you will have everything
-set up.
+.. NOTE::
+    Currently, your choice of region does not affect the price of the cluster.
+    This may change in future versions of CrateDB Cloud.
 
 
-Wizard step 1
--------------
+Configure cluster
+-----------------
 
-In the first step of the wizard, you are prompted for an organization, a
-project name, and a project region. Fortunately, the organization you have
-created earlier is already pre-selected for you. The region is also set by
-default, but can be changed using the region dropdown menu. All that is needed
-is to name the project within which your cluster will be deployed.
+Next, go through the cluster configuration process. On the left-hand side, you
+can choose a subscription plan. As you select each subscription plan, you will
+see that the values for CPU, RAM, and storage per node change accordingly in
+the middle panel that shows the node specification.
 
-.. image:: ../_assets/img/stripe-wizard-step1.png
-   :alt: CrateDB Cloud configuration wizard step 1
+For most subscription plans, the node specification also allows a further
+choice of tier: Basic, Pro, or Premium. Each higher tier doubles the values per
+node of the previous tier.
 
-Click *Next* to proceed.
+Finally, on the right the cluster scale overview shows the total values for the
+cluster. This is simply the number of nodes multiplied by the values of the
+subscription plan and tier. The cluster scale panel also allows you to increase
+the number of nodes you want in your cluster.
 
+.. image:: ../_assets/img/stripe-config.png
+   :alt: Cluster configuration panels
 
-Wizard step 2
--------------
+To sum up: the configuration of the cluster depends on the hardware values per
+node and the number of nodes in the cluster. The hardware values per node are
+determined by the choice of subscription plan and the choice of tier within the
+subscription plan. The number of nodes in the cluster is set in the cluster
+scale panel.
 
-In the next step, we come to the cluster itself. Here the wizard will ask you
-to name the cluster, as well as the username and the password that will
-subsequently be used to access the cluster via the unique cluster URL. The
-password must be at least 24 characters long; any characters are accepted,
-including special characters. If you want, click the *Auto-generate password*
-button to automatically generate a secure 24 character password (if it shows
-a password already, clicking again will generate a new one).
+Each of these choices will affect the price of the cluster, and it is possible
+for a subscription plan, a tier, or a number of nodes to be subject to a free
+or discounting promotion. You can always find the total price per hour and per
```

Comment:
Should this be _discounted_? discounting seems to read fine but not sure if it is correct, it might be

```suggestion
or discounted promotion. You can always find the total price per hour and per
```

Reply:
I was a bit conflicted about this... I wrote "discounted" at first but a discounted promotion seems a pleonasm, after all a promotion already implies at least a discount of some sort. But I did want to distinguish it from being free. So I went for this slightly ugly formulation. Not sure if it's the best one